experimental/desktop_photo_search/lib/src/widgets/split.dart

```diff
@@ -0,0 +1,194 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
This has a different author name than the others. Is it pulled from something else?

Reply:
Yup, it's @DaveShuckerow's code, and lifted and adapted from Flutter DevTool Widget Inspector, if memory serves.